docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,140 @@
+# Lagom Akka gRPC Quickstart with Scala
+
+Lagom is an open source framework (built on Akka and Play) for developing reactive microservice systems in Java or Scala.
+Akka gRPC is a toolkit for building streaming gRPC servers and clients on top of Akka Streams.
+
+This Guide will show you ho to use Akka gRPC as an alternate RPC library to communicate two microservices developed using Lagom. 
+
+## Downloading the example.
+
+The Lagom Akka gRPC Quickstart is a [GitHub repository](https://github.com/lagom/akka-grpc-lagom-quickstart-scala) that you can clone locally:
+
+```bash
+git clone https://github.com/lagom/akka-grpc-lagom-quickstart-scala.git
+```
+
+## Running the example
+
+Using gRPC in Lagom requires adding a Java Agent to the runtime. In order to handle this setting we provide a script that will
+download the ALPN Java Agent and start an interactive `sbt` console properly set up. Use the `ssl-lagom` 
+script:
+
+```
+./ssl-lagom
+```
+
+The first time you run the script it will take some time to resolve and download some dependencies. Once 
+ready you'll be at the `sbt` console. Use the `runAll` command to start the Lagom Akka gRPC Quickstart: 
+
+```
+sbt:akka-grpc-lagom-quickstart-scala> runAll
+```
+
+The `runAll` command starts Lagom in development mode. Once all the services are started you will see Lagom's start message:
+
+```
+...
+[info] Service hello-proxy-impl listening for HTTP on 127.0.0.1:54328
+[info] Service hello-proxy-impl listening for HTTPS on 127.0.0.1:65108
+[info] Service hello-impl listening for HTTP on 127.0.0.1:65499
+[info] Service hello-impl listening for HTTPS on 127.0.0.1:11000
+[info] (Services started, press enter to stop and go back to the console...)
+```  
+
+As soon as you see the message `[info] (Services started, press enter to stop and go back to the console...)` you 
+can proceed. On a separate terminal, try the application:
+
+```bash
+$ curl http://localhost:9000/proxy/rest-hello/Alice
+Hi Alice!
+$ curl http://localhost:9000/proxy/grpc-hello/Steve
+Hi Steve! (gRPC)
+```
+
+
+## Application Structure
+
+This application is built with two Lagom services (`hello` and `hello-proxy`) exposing the following endpoints:
+
+```
+GET /proxy/rest-hello/:id    # served by hello-proxy-service (HTTP-JSON)
+GET /proxy/grpc-hello/:id    # served by hello-proxy-service (HTTP-JSON)
+GET /api/hello/:id           # served by hello-service (HTTP-JSON)
+```
+
+And also 
+
+```
+  /helloworld.GreetingsService/sayHello   # served by hello-service (gRPC)
+```  
+
+None of the service include any logic to keep this quickstart as simple as possible. While the 
```

Comment:
"None of the service include any logic to keep this quickstart as simple as possible."

Maybe rewrite to:

"We want to show how to use gRPC for service communication, then the services are as simple as possible, with no other features."

Reply:
By the way, here is be a good place to show the `.proto` file and give some necessary explanation about it.

Reply:
> By the way, here is be a good place to show the .proto file and give some necessary explanation about it.

I wanted to keep `index.md` free of code. I think `index.md` could instruct how to obtain and exercise the app. Then further pages describe code in detail.

Reply:
So, IIUC, there should be new pull requests later to add more detailed docs, right?

Reply:
Small PRs. ;-)